src/OrderTransfer/order_transfer.go

```diff
@@ -0,0 +1,55 @@
+package ordertransfer
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/HyperloopUPV-H8/Backend-H8/OrderTransfer/domain"
+)
+
+type OrderTransfer struct {
+	OrderWebAdapterChannel chan OrderWebAdapter
+}
+
+func New() OrderTransfer {
+	return OrderTransfer{
+		OrderWebAdapterChannel: make(chan OrderWebAdapter),
+	}
+}
+
+func (orderTransfer OrderTransfer) Invoke(sendOrder func(order domain.Order)) {
+	go func() {
+		for orderWebAdapter := range orderTransfer.OrderWebAdapterChannel {
+			order := getOrder(orderWebAdapter)
+			sendOrder(order)
+		}
+	}()
+}
+
+func getOrder(orderWA OrderWebAdapter) domain.Order {
+	id, err := strconv.Atoi(orderWA.Id)
+
```

Comment:
Why do you need to pass to int, can't you pass directly to uint16?

Reply:
I can't do it directly. Atoi returns an int and another function called ParseInt returns a int64. In both cases I have to cast to uint16